### PR TITLE
BUGFIX: SD-90 remove the top number on yaxis of heatmap

### PR DESCRIPTION
**Pull request by lanhq**

### CODE REVIEW

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/979/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/510/ : FAILURE in 3m 09s

**Comment by lanhq** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/981/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/511/ : SUCCESS in 1m 44s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/280/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/980/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/126/ : UNSTABLE in 9m 02s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/281/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/982/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/127/ : SUCCESS in 8m 34s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1021/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/550/ : SUCCESS in 3m 57s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1029/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/559/ : SUCCESS in 4m 00s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/304/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1030/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/140/ : SUCCESS in 10m 07s

File: src/components/visualizations/chart/HighChartsRenderer.tsx [hunk 1]
```diff
@@ -53,9 +54,17 @@ function updateAxisTitleStyle(axis: Highcharts.AxisOptions) {
 }
 
 function setStartOrEndOnTick(axis: Highcharts.AxisOptions, chartOptions: any) {
-    const startOrEndOnTick = shouldStartOrEndOnTick(chartOptions);
-    set(axis, 'startOnTick', startOrEndOnTick);
-    set(axis, 'endOnTick', startOrEndOnTick);
+    const unDefinedStartOnTick = isUndefined(axis.startOnTick);
```
Comment (ivan-nejezchleb) [inline src/components/visualizations/chart/HighChartsRenderer.tsx]: fix typo - should be "undefined..."

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1050/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/573/ : SUCCESS in 9m 57s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/206/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/581/ : SUCCESS in 1m 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/313/ : SUCCESS in 7m 58s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/101/ : SUCCESS in 11m 42s

**Pull request merged**